app.py

```diff
@@ -322,9 +310,7 @@ def update_data_object(data_object_id):
         return Response({'msg': 'Please add a data_object to the body of your request'},
                         status_code=400)
 
-    new_aliases = filter(
-        lambda x: x not in data_object['aliases'],
-        update_data_object['aliases'])
+    new_aliases = [x for x in update_data_object['aliases'] if x not in data_object['aliases']]
     # if len(new_aliases) == 0:
```

Comment:
I would delete the code that's commented out.

Reply:
Fixed in acd9ca8!